### PR TITLE
Feat miscellaneous changes

### LINKED ISSUE #15: Todo: Miscellaneous Changes.
1) Fixing certain conditions while adding to wishlist.
2) Adding toasts.
3) Other minor UI changes

### PR DESCRIPTION
**Pull request by coldpigli**
All P0 features accounted for.

### CODE REVIEW

**Review by krishanu2911 — APPROVED** [review]
Well written code no changes.
Nicely written addtocart function short and understandable👍🏻

File: src/pages/Wishlist.jsx [hunk 1]
```diff
@@ -31,7 +32,12 @@ const Wishlist = () => {
       </div>
       </div>
       :
-      <h1>You need to login to access wishList</h1>
+      <div className="children-middle wrap">
+        <h1 className="txt-center gap-r30">You need to login to access Wishlist</h1>
+        <div>
+          <Link to="/login" className="btn btn-primary">Go To Login</Link>
+        </div>
+      </div>
```
Comment (kjais1720) [inline src/pages/Wishlist.jsx:40]: you can make a separate component for this and then use it in the cart page as well

**Review by kjais1720 — APPROVED** [review]
Everything looks good, just a small change suggested

File: src/pages/Wishlist.jsx (hunk 1 above)
Comment (coldpigli) [reply src/pages/Wishlist.jsx:40]: will do. thanks

**Pull request merged**